### ISSUE TITLE
[$50][MSFT-84] [Edge] The NVDA still read item name when user try to mouse hover outside area of them

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2795]
**Steps to Reproduce**
1. Navigate to https://accounts.topcoder.com/member/registration
2. Try to mouse hover outside area of any text-box or button

**Expected Result**
The NVDA should not read item name when user try to mouse hover outside area of them

**Actual Result**
The NVDA still read item name when user try to mouse hover outside area of them

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Edge

**WCAG Category**: 3. Understandable
**WCAG Level**: A
**WCAG Success Criterion**: 3.2.1 On Focus
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: 

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F13

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096275 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096275 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096275 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097880 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097880 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
@sushilshinde @Oanh-and-only-Oanh @codeMinter unable to reproduce.
Device: Desktop/Laptop
Operating System: Windows
Browser: Edge
Screen Reader Used: NVDA

Test URL - https://accounts.topcoder.com/member/registration

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097880 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
QA Reviewer: Confirm if this can be repoed or not

**Comment by drasticdpk** [comment]
HI @romitgit @crazyk07 Here I am able to replicate the issue with NVDA Version **2019.1.1**
Here is the reference video.
https://monosnap.com/file/XUDwtjNC88IOS3zmLRs0NAOpn9Qh42

**Comment by lakshmiathreya** [comment]
As per consensus from stakeholders, we are focusing exclusively on Chrome at this time. Putting this Issue on hold since it work on Chrome. fyi - @nkumar-topcoder @sushilshinde @drasticdpk @tosha5252

**Comment by crazyk07** [comment]
Closing issue and created another issue to address the Edge Browser.

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
